DXRBalance/DeusEx/Classes/BalancePlayer.uc

```diff
@@ -19,6 +19,10 @@ function TakeDamage(int Damage, Pawn instigatedBy, Vector hitlocation, Vector mo
             AddDamageDisplay('NanoVirus', vect(0,0,0));
             SetDamagePercent(1);
         }
+    } else if (damageType=='Shot'){
+        if (WineBulletsActive()){
```

Comment:
we might want to make WineBullets more efficient? maybe DXRandoCrowdControlEffects should just set a boolean on the player?

Reply:
Added a check for Crowd Control being active in the game for Wine Bullets and the Damage Multiplier, which should at least save some cycles for someone playing a normal game